tasks/notify.py

```diff
@@ -445,54 +447,128 @@ def update_statistics(job_executions: PipelineRuns):
             cumulative_alerts[job_name] = [job for job in job_executions.jobs[job_name].jobs_info if job.failing]
 
     return {
-        'consecutive': ConsecutiveJobAlert(consecutive_alerts),
-        'cumulative': CumulativeJobAlert(cumulative_alerts),
+        'consecutive': consecutive_alerts,
+        'cumulative': cumulative_alerts,
     }, job_executions
 
 
-def send_notification(ctx: Context, alert_jobs):
-    message = alert_jobs["consecutive"].message(ctx) + alert_jobs["cumulative"].message()
-    message = message.strip()
+def send_notification(ctx: Context, alert_jobs, jobowners=".gitlab/JOBOWNERS"):
+    def send_alert(channel, consecutive: ConsecutiveJobAlert, cumulative: CumulativeJobAlert):
+        message = consecutive.message(ctx) + cumulative.message()
+        message = message.strip()
 
-    if message:
-        send_slack_message("#agent-platform-ops", message)
+        if message:
+            send_slack_message(channel, message)
```

Comment:
Maybe not relevant for this PR, but I noticed it'd be really useful to have smth like a feed in a platform channel with the link of the sent messages on Slack. That way we'd be able to monitor what's happening.

Reply:
Thanks ! 😘